vector/src/main/res/values/colors.xml

```diff
@@ -3,6 +3,7 @@
 
     <!-- main app colors -->
     <color name="vector_green_color">#4D3F7E</color>
+    <color name="vector_white_color">#ffffff</color>
```

Comment:
Why this added color is prefixed with `vector_ ...`?
What do you think to name it `tchap_action_bar_text_color`?